filament/backend/src/vulkan/VulkanHandles.cpp

```diff
@@ -600,8 +601,9 @@ VulkanTexture::VulkanTexture(VulkanContext& context, SamplerType target, uint8_t
 
     if (any(usage & (TextureUsage::COLOR_ATTACHMENT | TextureUsage::DEPTH_ATTACHMENT))) {
         auto transition = [=](VulkanCommandBuffer commands) {
+            const uint32_t layers = target == SamplerType::SAMPLER_2D_ARRAY ? depth : 1;
```

Comment:
The name of the variable "depth" is confusing here.

Reply:
I'll add a clarification comment.